frontend/web/src/app/services/physical-print/physical-print.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import {Headers, Http, Response, ResponseOptions} from '@angular/http';
+import {Observable} from 'rxjs';
+import {HttpClientService} from '../http/http-client.service';
+import {HttpClient} from '../http/http.client';
+import {PhysicalPrint} from "../../model/physical-print";
+
+@Injectable()
+export class PhysicalPrintService {
+  private endpoint: string = HttpClient.physicalPrintUrl;
+
+  constructor(private http: Http, private client: HttpClientService) {}
+
+  getPhysicalPrint(id: number): Observable<PhysicalPrint> {
+    return this.client.get(this.endpoint + '/' + id)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)